changes/ocd-4437.sql

```diff
@@ -0,0 +1,30 @@
+create table if not exists openchpl.updated_listing_status_report (
+    id bigserial not null,
+    certified_product_id bigint not null,
+    report_day date not null,
+    criteria_require_update_count int,
+    days_updated_early int,
+    chpl_product_number text not null,
```

Comment:
Do we need to save this if we are saving the certified product id? Asking because this (and the columns below - product, version, developer, certification status) can change over time. If there is a reason to save this data "as it was" at the time the report data generator ran then I'm good with this. But if we wanted the emailed report to reflect this data "as it is now" then maybe we don't save all this stuff? Thoughts?

Reply:
Discussed with Todd and learned that the report is intended to be a snapshot of the data as it was. Resolving the comment.